package.json

```diff
@@ -246,6 +259,15 @@
       }
     ],
     "debuggers": [
+      {
+        "type": "NAMESPACE(node-terminal)",
+        "label": "Javascript Auto-Attaching Terminal",
+        "configurationAttributes": {
+          "launch": {
+            "properties": {}
```

Comment:
I'm going to fill this out when I put in a PR (tomorrow/friday) to generate parts of the package.json, I don't want to copy all the common node launch stuff for a third time...